project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 0.13.12
\ No newline at end of file
+sbt.version = 1.0.1
```

Comment:
@longshorej - I think SBT `1.0.2` is out. I think we should try to be on this version, unless there's a specific why you think we shouldn't be.

Reply:
Nice - thanks!